src/components/avatar/avatar.mod.css

```diff
@@ -0,0 +1,53 @@
+:root {
+  --size-s: 26px;
+  --size-m: 48px;
+  --size-l: 100px;
```

Comment:
This is note for designers... Usually our design system should provide us with these sizes. We should not define "arbitrary" sizes for specific components.

I'm okay with it for now, as we can work on this incrementally. However I want us to be aware that what we're doing here is not the way it's supposed to be done.

cc @commercetools/design-team